bittide-shake/bin/Shake.hs

```diff
@@ -140,7 +134,7 @@ findProjectRoot = goUp =<< getCurrentDirectory
 
 data Target = Target
   { -- | TemplateHaskell reference to top entity to synthesize
-    targetName :: TH.Name
+    targetName :: String
```

Comment:
```suggestion
    targetName :: TargetName
```

with a type alias:

```haskell
-- | Fully qualified name to a function. E.g. @Bittide.Foo.topEntity@.
type TargetName = String
```

That makes the type signatures a bit easier to read (and establishes soft rules for what its contents should look like).